fiat-api/src/main/java/com/netflix/spinnaker/fiat/shared/FiatPermissionEvaluator.java

```diff
@@ -206,4 +210,8 @@ private boolean permissionContains(UserPermission.View permission,
   public boolean isAdmin() {
     return true; // TODO(ttomsu): Chosen by fair dice roll. Guaranteed to be random.
   }
+
+  private boolean isEnabled() {
+    return dynamicConfigService.isEnabled("fiat", configProps.isEnabled());
```

Comment:
did a hack verification that the evaluator would start to short-circuit as expected when fiat was disabled